components/ingest-service/server/jobs.go

```diff
@@ -1,124 +1,280 @@
 package server
 
 import (
-	"github.com/chef/automate/components/ingest-service/backend"
-	"github.com/chef/automate/components/ingest-service/config"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
```

Comment:
There are two "logrus" import lines.

Reply:
Ah, thanks, must have use `logrus` rather than `log` down below out of habit.